jupyter_server_proxy/handlers.py

```diff
@@ -363,7 +364,85 @@ async def proxy(self, host, port, proxied_path):
                 body = b""
             else:
                 body = None
+        accept_type = self.request.headers.get('Accept')
+        if accept_type == 'text/event-stream':
+            return await self._proxy_progressive(host, port, proxied_path, body)
+        else:
+            return await self._proxy_normal(host, port, proxied_path, body)
+    
+    async def _proxy_progressive(self, host, port, proxied_path, body):
+        # Proxy in progressive flush mode, whenever chunks are received. Potentially slower but get results quicker for voila
+
+        client = httpclient.AsyncHTTPClient()
+
+        # Set up handlers so we can progressively flush result
+
+        headers_raw = []
+
+        def dump_headers(headers_raw):
+            for line in headers_raw:
+                r = re.match('^([a-zA-Z0-9\-_]+)\s*\:\s*([^\r\n]+)[\r\n]*$', line)
+                if r:
+                    k,v = r.groups([1,2])
+                    if k not in ('Content-Length', 'Transfer-Encoding',
```

Comment:
This is a different list than the one in line 531. Is there a specific reason? If not, I think we should move this to a constant at the top of the file and reference it from both places so it doesn't get out of date.